server_admin/topics/threat/host.adoc

```diff
@@ -0,0 +1,40 @@
+
+=== Host
+
+{project_name} uses the request URL for a number of things. For example the URL sent in password reset emails.
+
+By default the request URL is based on the `Host` header and there is no check to make sure this URL is the valid and
+correct URL.
+
```

Comment:
You need a couple commas. In line 4, after "For example," and in line 6 after "By default". Also in line 9 after "invalid host headers."